File: android/src/main/java/expo/modules/appgrouppath/ExpoAppgroupPathModule.kt
package expo.modules.appgrouppath

import expo.modules.kotlin.modules.Module
import expo.modules.kotlin.modules.ModuleDefinition

class ExpoAppgroupPathModule : Module() {
  override fun definition() = ModuleDefinition {
    Name("ExpoAppgroupPath")

    Function("getPath") {
      "Unsupported"
    }
  }
}

File: ios/ExpoAppgroupPathModule.swift
import ExpoModulesCore
import Foundation

public class ExpoAppgroupPathModule: Module {
  public func definition() -> ModuleDefinition {
    Name("ExpoAppgroupPath")
      
    Function("getPath") { (group: String) -> String? in
      FileManager.default.containerURL(forSecurityApplicationGroupIdentifier: group)?.absoluteString
    }
  }
}
